frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
+  border-radius: 8px;
+  padding: 8px;
```

Comment:
```suggestion
  margin: 8px;
  padding: 12px;
```

Reply:
updated the hover states for notification and avatar. the 12px padding still looks to big in relation to the header (isolated it looks fine) so kept it at 8px

Reply:
12 looks good to me. We can always adjust when we move the search bar and remove the hamburger. I think for desktop we have the space though :)